src/sentry/api/endpoints/group_events_latest.py

```diff
@@ -32,7 +32,11 @@ def get(self, request, group):
 
         :pparam string group_id: the ID of the issue
         """
-        event = group.get_latest_event()
+
+        requested_environments = set(request.GET.getlist('environment'))
```

Comment:
Nbd, but it's slightly better to use `sentry.api.helpers.environments:get_environments` since it'll validate that they actually exist.

Reply:
I did this to save a database query - do we prefer to validate?

Reply:
Not sure how much it really matters, but we do it in other places.

Reply:
The result would be exactly the same either way

Reply:
It lets us give the user slightly better error messages, rather than just no data if they enter an invalid environment. I don't feel that strongly either way, just wanted to mention it.